pyphs/numerics/simulations/simulation.py

```diff
@@ -353,39 +353,46 @@ def process(files):
         # close_files(files)
 
     def _process_cpp(self):
-
-        # build simu.cpp
+        
+                # build simu.cpp
         simu2cpp(self)
 
         # go to build folder
         os.chdir(self.cpp_path)
 
         # execute the bash script
-        self.system_call('./run.sh')
-
-        # Check if an error occured outside python (bash)
-        self._check_cpp_runtime_errors()
+        # self.system_call('./run.sh')
         
-        # Removes the error file if everything went well
-        os.remove('stderr')
+        # Commands
+        cmd = {
+               'Bbuild': '%s . -Bbuild' % self.config['cmake'],
```

Comment:
You can even be more explicit in the keys of the cmd dictionary. For example:
Bbuild -> build_tree
build -> compile_src